lib/phone/be.ex

```diff
@@ -3,7 +3,7 @@ defmodule Phone.BE do
 
   use Helper.Country
 
-  def regex, do: ~r/^(32)()(.{9})/
+  def regex, do: ~r/^(32)0{0,1}(.{8,9})/
```

Comment:
So the change to be made is something like this
```suggestion
  def regex, do: ~r/^(32)()(.{8,9})/
```
and you could use the suggestion that i made locally, because the leading zero is the trunk code for the country and it's not part of the international representation of the number according to the link you sent:
```
From outside Belgium, a caller would dial their international call prefix (typically 00 in Europe and 011 in North America), followed by 32 (the country code for Belgium), then the area code minus the trunk code '0', and finally the local number.
```

Reply:
Can we use this one ` ~r/^(32)(0{0,1}.{1,3})(.{6,7})/`

Reply:
that's actually incorrect because the area code will have only 3 characters if the first one is 4 it would be something like this `~r/^(32)(4..|.{1,2})(.{6,7}}/` and locally you should use that example:
```elixir
def your_function("0" <> phone_number), do: Phone.parse(phone_number, :be)
def your_function(phone_number), do: Phone.parse(phone_number, :be)
```
with this code if you have the trunk code or not in the number, it will work.

I'm against including trunk code in this library, because the assumption here is that we're parsing international numbers. If i started supporting this on Belgium, i should do that for all countries and there are some cases that the trunk code is 1 or something weird that would make validating the number difficult.

Reply:
In our service we find people in linkedin and this one from Belgium shared his phone both with +32 and leading 0. I can't use functions that you share b/c for this I need to parse phone number first and remove +32 from number and only after that remove leading 0. It's seems people don't care what they share in their profiles - international phone or local. E.g. in russia it's typical when people shares their numbers with 8 instead of +7.

Reply:
Btw I already have local patch phone normalization for inserting leading 0 for ZA phone numbers:
```elixir
  # Fix south africa phone numbers validation.
  # They omit the leading 0 after the country code
  # and we add it to pass the validation
  defp fix_za_phone_numbers(string) do
    match = Regex.named_captures(~r/^(?<code>27)(?<number>\d{9})$/, string)

    if match do
      match["code"] <> "0" <> match["number"]
    else
      string
    end
  end
```

I'd rather remove this code from my project and move it to phone lib.